ansible/playbook/tasks/update.yml

```diff
@@ -183,6 +191,7 @@
           - mariadb-libs # https://jira.percona.com/browse/PMM-5215
           - logrotate # https://jira.percona.com/browse/PMM-7627
           - screen
+          - yum-cron
```

Comment:
Why do we need it? We don't use yum-cron job anymore

Reply:
This is removing, not adding. When you upgrade with the button, yum-cron is still installed on the server. Hence we want to remove it.